### PR TITLE
Add Mobile Hub section

### PR DESCRIPTION
**Pull request by ceich**
I added a Mobile Hub section with a new TOC entry.

There are some whitespace diffs elsewhere in the file that Atom made on its own. Sorry!

### CODE REVIEW

**Comment by ceich** [comment]
@QuinnyPig how's this look?

File: README.md [hunk 1]
```diff
@@ -1323,7 +1324,7 @@ DynamoDB
 -	DynamoDB can be used [as a simple locking service](https://gist.github.com/ryandotsmith/c95fd21fab91b0823328).
 -	DynamoDB indexing can include **primary keys**, which can either be a single-attribute hash key or a composite hash-key range. You can also query non-primary key attributes using [**secondary indexes**](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/SecondaryIndexes.html).
 -	**Data Types:** DynamoDB supports three [data types](http://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBMapper.DataTypes.html) – **number**, **string**, and **binary** – in both scalar and multi-valued sets. DynamoDB can also support [**JSON**](https://aws.amazon.com/blogs/aws/dynamodb-update-json-and-more/).
--	As of late 2017, DynamoDB supports both [global tables](https://aws.amazon.com/dynamodb/global-tables/) and [backup / restore functionality](https://aws.amazon.com/dynamodb/backup-restore/). 
+-	As of late 2017, DynamoDB supports both [global tables](https://aws.amazon.com/dynamodb/global-tables/) and [backup / restore functionality](https://aws.amazon.com/dynamodb/backup-restore/).
```
Comment (QuinnyPig) [inline README.md]: ...what changed? Have I gone blind?

File: README.md [hunk 2]
```diff
@@ -771,7 +772,7 @@ S3
...
+-	🔸**Incomplete multi-part upload costs:** Incomplete multi-part uploads accrue [storage charges](http://docs.aws.amazon.com/AmazonS3/latest/dev/mpuoverview.html#mpuploadpricing) even if the upload fails and no S3 object is created. [Amazon](http://docs.aws.amazon.com/AmazonS3/latest/dev/mpuoverview.html#mpu-abort-incomplete-mpu-lifecycle-config) ([and](http://www.deplication.net/2016/06/aws-tip-save-s3-costs-with-abort.html) [others](https://www.sumologic.com/aws/s3/s3-cost-optimization/)) recommend using a lifecycle policy to clean up incomplete uploads and save on storage costs. Note that if you have many of these, it may be worth investigating whatever's failing regularly.
```
Comment (QuinnyPig) [inline README.md]: Is this whitespace, or is Github messing with me?

**Comment by QuinnyPig** [comment]
Other than not seeing a couple of changes, I like it quite a bit. @jlevy / @thanos?

**Comment by ceich** [comment]
Sorry about the extra diffs, Atom is aggressive about eliminating trailing whitespace and adding final newlines (and one of the diffs was a total mystery: maybe a strange quark?).

**Comment by ThanosBaskous** [comment]
Great work, @ceich!

**Pull request merged**